frontend/javascripts/oxalis/model/sagas/save_saga.js

```diff
@@ -91,6 +97,14 @@ type racedActionsNeededForUndoRedo = {
   redo: ?RedoAction,
 };
 
+// Better version: Let the undo stack / batch know that bucket data is not yet fetched by the backend.
```

Comment:
This comment seems a little out of place here. Also, the "Better version:" beginning can be dropped I'd say. Maybe around line 240 would be more suitable to explain this (or above the `compressBucketAndAddToUndoBatch` function)?

Reply:
I simply removed the comment, as these were only notes from the call with philipp on how to solve this bug.

And I think you reviewed an old version here, as I can no longer find the comment :eyes: :swan:

Reply:
You did, but it seems like you didn't push this change until the 16.07. which is why I saw the "old" code during my review. In any case, all good :)